### PR TITLE
Add aide

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:
- add aide to releases to build

## security considerations
None

### CODE REVIEW

File: releases/releases.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ releases:
   uri: https://github.com/cloud-gov/uaa-customized-boshrelease
   branch: master
 - name: admin-ui
-  uri: https://github.com/cloudfoundry-community/admin-ui-boshrelease
+  uri: https://github.com/jmcarp/admin-ui-boshrelease
```
Comment (ccostino) [inline releases/releases.yml:28]: Out of curiosity, why the switch to Josh's fork of this?

Comment (bengerman13) [reply releases/releases.yml:28]: this is reverting a change I'd made previously - `master` was even on the forks, and I didn't notice that we're actually building `deploy`

Comment (ccostino) [reply releases/releases.yml:28]: Ah, which does have a couple of modifications not in upstream it looks like?

Comment (bengerman13) [reply releases/releases.yml:28]: correct. There should be a PR soon to nuke this release in any case

**Pull request merged**